webapp/templatetags/resolve.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag()
+def resolve(lookup, target):
+    try:
+        return target[lookup]
+
+    except (IndexError, KeyError):
```

Comment:
Why do you want to suppress these errors? If someone messes up their template by requesting a key that doesn't exist, might not an `KeyError`, leading to a 500 response, be the appropriate signal to get them to fix their code?

Reply:
The problem here was, that not all articles have categories. I think thats why I included this catch all.
I could do a check in the template instead though, and let this function crash :)

Reply:
Yes I think that would be better. Otherwise people are likely to silently add mistakes they never realise about.